core/View/HtmlReportEmailHeaderView.php

```diff
@@ -77,7 +78,13 @@ public static function assignCommonParameters(View $view)
         $view->isCustomLogo = $customLogo->isEnabled() && CustomLogo::hasUserLogo();
         $view->logoHeader = $customLogo->getHeaderLogoUrl($pathOnly = false);
 
-        $view->hasWhiteLabel = \Piwik\Plugin\Manager::getInstance()->isPluginLoaded('WhiteLabel');
+        $view->hasWhiteLabel = \Piwik\Plugin\Manager::getInstance()->isPluginLoaded('WhiteLabel')
+            && \Piwik\Plugin\Manager::getInstance()->isPluginActivated('WhiteLabel')
+            && \Piwik\Plugin\Manager::getInstance()->isPluginInFilesystem('WhiteLabel');
+
+        $view->idSite = Common::getRequestVar('idSite', false);
```

Comment:
Not sure about those 3 lines. Are they needed @diosmosis ? For what? Cause the current set `idSite` in the request might not be related to any sent email. Same for period or date. Sometimes  there eg hooks that listen to something like a plugin update, and then send an email about something totally else.

Reply:
These three params are used in the dashboard link in the email header. Some emails had it, some didn't. Thought I'd add it to all if the params were available.

Reply:
Wouldn't we want to link the the default report?

Reply:
This is how the code worked before, but I can change it to link to what's configured in settings.

Reply:
Nvm, it already does this (by linking to CoreHome.index).

Reply:
Wouldn't it need to link to `CoreHome.redirectToCoreHomeIndex` (requires no site, period, date, ... AFAIK)? I guess the only time we wouldn't want this is when a report is sent for a specific site. In all cases might make sense to load the default report and default time to load.

Reply:
Yes, would need to link to `redirectToCoreHomeIndex` or I guess to omit the action. I can make this change, except when a specific site is set.